libsignal-service/src/cipher.rs

```diff
@@ -206,7 +206,7 @@ where
             Type::UnidentifiedSender => {
                 let SealedSenderDecryptionResult {
                     sender_uuid,
-                    sender_e164,
+                    sender_e164: _,
```

Comment:
Something tells me that this is also removed with a protobuf update?

Reply:
This is a [concrete type from `libsignal`](https://github.com/signalapp/libsignal/blob/main/rust/protocol/src/sealed_sender.rs#L158) which has the phone number as `Option<String>`.